### DISCUSSION TITLE
Custom Header: Need Help Modifying Header to Display Site Name and Tagline Stacked Vertically

### DISCUSSION THREAD

**Discussion by ghost** [nunocoracao/blowfish#1860] [Q&A]
Hello,

I am trying to modify the header of my site so that the site name and tagline appear one above the other, like this:

Site Name
Tagline

As a complete beginner, I attempted to use ChatGPT for guidance, but I haven't been able to create a custom header that matches what I want. I've spent several days going through the documentation, but I still can't figure out how to make this change.

Is there a way to accomplish this? Any help would be greatly appreciated!

**Comment by ghost** [comment]
Hello there, could you please, point me to the source, article, discussion, etc. where I can learn and solve this issue? I am missing something, and I can't figure out what. I saw different websites built with bloowfish, with headers changed, containing logos or names of the website. Havent seen website and tagline one above the other though.. There is probably the most obvious thing that im missing here. If someone find some time to respond i would apreciate the effort. Thank you in advance! Have a nice day!

**Comment by R0gue-one** [comment]
@tatamata500 Hey! I hope Im not too late. I think what you need is [this](https://blowfish.page/docs/homepage-layout/#custom-layout).

In this just make `header` folder instead of `home` and what you might want to do now is copy [this file](https://github.com/nunocoracao/blowfish/blob/main/layouts/partials/header/basic.html) this is the html file of header and you can now change the header as you like (I belive you will be looking for navbar with `Site.Title`). Also don't forget to change the header style in `params.toml` to the new custom filename.

If you have any queries let me know I'll try by best to help

**Comment by ghost** [comment]
Thank you for the help! 

In the meantime, I did some changes to attain what I wanted. I droped the screenshot of the site name and tagline one above the other, and it works. It is actually a "logo".. it behaves like a button to homepage.

**Reply by R0gue-one** [reply]
Yeah! that's actually a creative way to solve it
